src/ott/math/utils.py

```diff
@@ -37,6 +43,41 @@ def safe_log(  # noqa: D103
   return jnp.where(x > 0., jnp.log(x), jnp.log(eps))
 
 
+@functools.partial(jax.custom_jvp, nondiff_argnums=[1])
+def norm(x, ord=2.0):
```

Comment:
I think this function should accept all arguments of `jnp.linalg.norm` - `axis` and `keepdims` as well.

Reply:
Please also add type hints.

Reply:
Not completey sure we need the `axis` and `keepdims`, these tend to help mostly when computing matrix norms, which haven't appeared yet in our use-case as a ground cost. But will add, as long as this does not cause confusions or difficulties!